inc/zoo/swar/SWAR.h

```diff
@@ -161,6 +213,12 @@ struct SWAR {
     T m_v;
 };
 
+template <int NBits, typename T, typename Arg>
+SWAR(Literals_t<NBits, T>, const Arg (&values)[SWAR<NBits, T>::Lanes]) -> SWAR<NBits, T>;
```

Comment:
```suggestion
SWAR(Literals_t<NBits, T>, const Arg (&values)[SWAR<NBits, T>::Lanes])
-> SWAR<NBits, T>;
```

Reply:
```suggestion
SWAR(
    Literals_t<NBits, T>, const Arg (&values)[SWAR<NBits, T>::Lanes]
) -> SWAR<NBits, T>;
```

Reply:
```suggestion
SWAR(
    Literals_t<NBits, T>,
    const Arg (&values)[SWAR<NBits, T>::Lanes]
) -> SWAR<NBits, T>;
```

Reply:
You can also create a new line and indentation level each `<` of each template.
I would not mind any of those.
And you can use K&R, as in
```
void function()
{
    // ...
} 
```